core/src/main/java/com/statful/client/core/StatfulClientImpl.java

```diff
@@ -93,6 +93,17 @@ public final SenderFacade gauge(final String metricName, final Integer value) {
     }
 
     @Override
+    public final SenderFacade metric(final String metricName, final Integer value) {
+        SenderAPI senderAPI = MetricsSenderAPI.newInstance(this).with()
+                .configuration(configuration)
+                .aggFreq(configuration.getDefaultAggregationFreq())
```

Comment:
I'm not following, if I understood correctly this would be a simplified method overload of the put method with only the metric name and value. The put method can only have a default aggregation frequency if any aggregation is specified.

This is because default aggregations only make sense on a per metric type basis... ie. timer = avg,p90,count; counter = sum,count; etc. and we don't really know what type of metric the user wants to send.